multi-agent/lib/mas/core/element_deps.py

```
@@ -1,10 +1,11 @@
 from __future__ import annotations
 
 from dataclasses import dataclass, field
-from typing import Optional
+from typing import Callable, Optional
```

Ruff (0.15.15)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)